### DISCUSSION TITLE
v3.1.9-beta.0 && beta.1

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2375] [Q&A] [answered]
Was wurde geändert?

**Comment by Zefau** [comment, marked as answer]
Das Changelog steht ab v3.1.8 jetzt immer im Release anhand der bearbeiteten Tickets. Im Fall der beta.0 und beta.1 habe ich Änderungen im Core gemacht für die es kein Ticket gab, daher ist der Changelog leer.
